prov/tcp/src/tcpx_init.c

```diff
@@ -98,6 +108,22 @@ static void tcpx_init_env(void)
 		port_range.low  = 0;
 		port_range.high = 0;
 	}
+
+	fi_param_get_int(&tcpx_prov, "tx_size", &tx_size);
+	if (tx_size < 0) {
+		FI_WARN(&tcpx_prov, FI_LOG_EP_CTRL,"User provided "
+			"tx context size invalid. Ignoring. \n");
```

Comment:
converting to size_t will allow dropping the warnings.

Reply:
Fixed. Sorry, I didn't pay attention to the compilation output.
I'm not compiling with "-Werror" but I probably should in order to catch these bugs sooner :)